tools/profiling/Hook.php

```diff
@@ -44,6 +44,44 @@ public static function coreCallHook($module, $method, $params)
             ]
         );
 
+        Profiler::getInstance()->interceptModule(
+            [
+                'module' => $module->name,
+                'method' => $method,
+                'time' => microtime(true) - $timeStart,
+                'memory' => memory_get_usage() - $memoryStart,
+            ]
+        );
+
+        return $result;
+    }
+
+    public static function coreRenderWidget($module, $registeredHookName, $params)
```

Comment:
Could you typeint this new method ?

Reply:
I don't think I can as there is a warning about method not compatible with parent :)

Reply:
If it extends, you can. If it was an implementation (interface), you can't.

Reply:
@Progi1984 

https://www.dropbox.com/s/mtm385bi96rgl5z/Zrzut%20ekranu%202021-05-26%20o%2015.39.48.png?dl=0